sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -401,6 +401,13 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
     }
   }
 
+  override def alterTableStats(tableDefinition: CatalogTable): Unit = withClient {
```

Comment:
I'm not sure we wanna support the second way to set properties. If users set them with ALTER TABLE, we should throw exception. cc @yhuai @gatorsmile what do you think?